general/envsetup/envsetup_test.go

```diff
@@ -0,0 +1,16 @@
+package envsetup
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestSetupInvitedUser(t *testing.T) {
```

Comment:
I can test it in cli's tests - cause if I will run the config command on this invalid base64 value it will fail (trying to ping artifactory in the end and fails all the config flow)
also can't generate from test's short lived accessToken a refreshToken. it can be tested only in integration tests.

Reply:
OK, so you're going to leave this test as is, and add an additional test on `jfrog-cli`?

Reply:
yes